Formula/f/fx-upscale.rb

```diff
@@ -0,0 +1,27 @@
+class FxUpscale < Formula
+  desc "Metal-powered video upscaling"
+  homepage "https://github.com/finnvoor/fx-upscale"
+  url "https://github.com/finnvoor/fx-upscale/archive/refs/tags/1.2.5.tar.gz"
+  sha256 "4ec46dd6433d158f74e6d34538ead6b010455c9c6d972b812b22423842206d8b"
+  license "CC0-1.0"
+
+  depends_on :macos
+  depends_on macos: :sonoma
+  uses_from_macos "swift"
```

Comment:
maybe?
```suggestion
  uses_from_macos "swift" => :build, since: :sonoma
```

Reply:
I didn't know if it was needed because it depends on Sonoma?

Reply:
There's some checks in the code that specifically look for Sonoma or later.